### PR TITLE
Virgonew 955

### PR DESCRIPTION
**Pull request by ChristopherWelte**
https://jira.lib.virginia.edu/browse/VIRGONEW-955

### CODE REVIEW

File: frontend/src/components/layout/SearchingOverlay.vue [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <template>
    <div v-show="searching" class="searching-overlay">
       <div class="searching-box">
-         <p>{{message}}</p>
-         <img src="../../assets/searching.gif">
+        <div class="book">
+          <div class="book-page"></div>
```
Comment (louffoster) [inline frontend/src/components/layout/SearchingOverlay.vue:5]: Why the book div and 3 empty book-page divs?

Comment (louffoster) [reply frontend/src/components/layout/SearchingOverlay.vue:5]: nevermind... i see why later on in the changes

**Review by louffoster — APPROVED** [review]
looks good

File: frontend/src/components/layout/SearchingOverlay.vue (hunk 1 above)
Comment (ChristopherWelte) [reply frontend/src/components/layout/SearchingOverlay.vue:5]: Argh, I forgot to clean out that commented code. 
Those empty divs are the "pages" of the book animation.

Comment (ChristopherWelte) [reply frontend/src/components/layout/SearchingOverlay.vue:5]: I don't have a current way to test this in Windows (I did on my old laptop - hoping to get VMware installed soon)
If this doesn't work in Windows we can roll back to using an animated gif.

**Pull request merged**